test/dynamo/test_higher_order_ops.py

```diff
@@ -2964,22 +3070,24 @@ def fn(x):
             """\
 class GraphModule(torch.nn.Module):
     def forward(self, L_x_ : torch.Tensor):
-        child = L_x_
+        arg = L_x_
 
-        _check_randomness_arg = torch._functorch.vmap._check_randomness_arg('error')
+        lazy_load_decompositions = torch._functorch.vmap.lazy_load_decompositions()
 
-        select = child.select(0, 0)
-        vmap_body_0 = self.vmap_body_0
-        vmap_proxy = torch.func.vmap(vmap_body_0, (0,), 0, 'error');  vmap_body_0 = None
-        call = vmap_proxy.__call__(child);  vmap_proxy = child = None
-        return (call,)
+        _saved_tensors_hooks_disable = torch._C._autograd._saved_tensors_hooks_disable("torch.func transforms don't yet support saved tensor hooks. Please open an issue with your use case.")
```

Comment:
To check: like vmap_increment_nesting, _saved_tensors_hooks_disable is also a context-manager-like thing, did we implement it as a ContextManager? (i.e. does Dynamo undo the state on a graph break inside vmap(f)(x))?

Reply:
@kshitij12345 implemented support for it in https://github.com/pytorch/pytorch/pull/104869